src/components/header.tsx

```diff
@@ -4,23 +4,33 @@ import Link from "next/link";
 import LoginButton from "~/components/login-button";
 import LogoutButton from "./logout-button";
 import { getServerSession } from "next-auth";
+import localFont from "next/font/local";
+import { cn } from "~/lib/utils";
 
+const myFont = localFont({ src: "./../app/fonts/AltivoMedium.otf" });
```

Comment:
Can we maybe make a `font.ts` file in `styles/` for instance that does something like the following so that we can re-use fonts more efficiently?

```javascript
import localFont from "next/font/local";

export const altivoFont = localFont({
  src: [
    {
      path: "<path>",
      weight: "400",
      style: "normal",
    },
    {
      path: "<path>",
      weight: "600",
      style: "normal",
    }
});

// Same for Benton Sans
```

And then import this in `header.tsx` and `layout.tsx`.